pymagicc/io.py

```diff
@@ -2095,3 +2119,106 @@ def get_generic_rcp_name(inname):
     except KeyError:
         error_msg = "No generic name for input: {}".format(inname)
         raise ValueError(error_msg)
+
+
+# TODO: move out of here in clean up
```

Comment:
Create an issue, do it or somehow make it clearer how this could be re-factored?